File: math/multiplication-table/timestab.java
/**
 * Copyright https://github.com/microwind
 * Multiplication Table
 */

class TimesTable {
    public static void printTable(int max) {
        for (int i = 1; i <= max; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.printf("%d×%d=%2d  ", j, i, i * j);
            }
            System.out.println();
        }
    }
    
    public static void main(String[] args) {
        System.out.println("Multiplication Table (9x9):");
        printTable(9);
    }
}

File: math/multiplication-table/timestab.js
/**
 * Copyright https://github.com/microwind
 * Multiplication Table
 */

function printTable(max) {
    for (let i = 1; i <= max; i++) {
        let line = '';
        for (let j = 1; j <= i; j++) {
            line += `${j}×${i}=${(i * j).toString().padStart(2)}  `;
        }
        console.log(line);
    }
}

function main() {
    console.log("Multiplication Table (9x9):");
    printTable(9);
}

main();
if (typeof module !== 'undefined') module.exports = { printTable };
